app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -146,6 +148,7 @@ md-input-group {
 md-toolbar {
   @include gamma;
   padding: 16px;
+  max-height: $toolbar-max-height;
```

Comment:
I don't think so, it might be ok to set the height on all toolbars. I'll change that